pom.xml

```diff
@@ -284,7 +283,7 @@
                                         <version>1.8.0</version>
                                     </requireJavaVersion>
                                     <requireMavenVersion>
-                                        <version>3.1.0</version>
+                                        <version>3.3.9</version>
```

Comment:
If you want to build using Java 8 and a relatively recent version of maven that's not the absolute latest, I think you should be able to.